src/Analyser/FileAnalyser.php

```diff
@@ -281,6 +290,17 @@ public function analyseFile(
 			unset($unmatchedLineIgnores[$fileKey]);
 		}
 
+		$fileErrors = array_filter($fileErrors, function (Error $error) use ($scope) : bool {
+			foreach ($this->ignoreErrorExtensionProvider->getExtensions() as $ignoreErrorExtension) {
+				if ($ignoreErrorExtension->ignore($error, null, $scope)) {
```

Comment:
Not sure what this covers, maybe we don't need this block at all.

Reply:
Hmm, I don't need it either, but if we _do_ want this in the future, we have an issue because the interface will then always have a Node, which for FileErrors won't be available. So what to do?

Reply:
There is also no scope, so I think it makes sense to remove it. Done.